examples/streaming_memory_usage.c

```diff
@@ -19,6 +19,7 @@
 #include <stdio.h>   /* printf */
 #define ZSTD_STATIC_LINKING_ONLY
 #include "zstd.h"
+#include "utils.h"
```

Comment:
Is this dependency really necessary ?

Generally, the examples should better be as self-contained as possible, depending only on standard libs and `libzstd`.

Reply:
I'll put up a separate PR to rename `utils.h` after these have landed.